Makefile

```diff
@@ -4,7 +4,8 @@ SUDO := $(shell docker version >/dev/null 2>&1 || echo "sudo")
 # https://github.com/opencontainers/image-spec/blob/master/annotations.md
 BUILD_DATE := $(shell date -u +%FT%TZ)
 
-ETLHASH=stellar/stellar-etl:$(shell git rev-parse --short=9 HEAD)
+DEFAULT_ETLHASH := stellar/stellar-etl:$(shell git rev-parse --short=9 HEAD)
+ETLHASH ?= $(DEFAULT_ETLHASH)
```

Comment:
curious: why this was split into two vars?

Reply:
To support both versions - local build and github action build. If we use `make docker-build` locally, it will use ETLHASH = DEFAULT_ETLHASH. Otherwise, for github action, ETLHASH is passed as override env variable from branch

https://github.com/stellar/stellar-etl/blob/af9eaa88b21d5402c9acf720558ff7c50cd38939/.github/workflows/build.yml#L28

Reply:
Ah, makes sense. Thanks 🙏